### PR TITLE
Update zh translation in mysql-wordpress-persistent-volume.md

### PR DESCRIPTION
**Pull request by wwgfhf**
Update zh translation in mysql-wordpress-persistent-volume.md.

### CODE REVIEW

**Comment by xichengliudui** [comment]
/lgtm
/approve
@wwgfhf thanks!

File: content/zh/docs/tutorials/stateful-application/mysql-wordpress-persistent-volume.md [hunk 1]
```diff
@@ -185,7 +185,7 @@ the name of the MySQL Service defined above, and WordPress will access the datab
 `WORDPRESS_DB_PASSWORD` environment variable sets the database password from the Secret kustomize generated. 
 -->
 
-以下 manifest 文件描述了单实例 WordPress 部署。WordPress 容器将网站数据文件位于`/var/www/html`的 PersistentVolume。`WORDPRESS_DB_HOST`环境变量集上面定义的 MySQL Service 的名称，WordPress 将通过 Service 访问数据库。`WORDPRESS_DB_PASSWORD`环境变量设置从 Secret kustomize 生成的数据库密码。
+以下 manifest 文件描述了单实例 WordPress 部署。WordPress 容器为网站数据文件将PersistentVolume挂载到了`/var/www/html`。`WORDPRESS_DB_HOST`环境变量上面定义了 MySQL Service 的名称，WordPress 将通过 Service 访问数据库。`WORDPRESS_DB_PASSWORD`环境变量设置从 Secret kustomize 生成的数据库密码。
```
Comment (xichengliudui) [inline content/zh/docs/tutorials/stateful-application/mysql-wordpress-persistent-volume.md]:
```suggestion
以下清单文件描述了单实例 WordPress 部署。WordPress 容器为网站数据文件将 PersistentVolume 挂载到了 `/var/www/html`。`WORDPRESS_DB_HOST` 环境变量上面定义了 MySQL Service 的名称，WordPress 将通过 Service 访问数据库。`WORDPRESS_DB_PASSWORD` 环境变量设置从 Secret kustomize 生成的数据库密码。
```

**Comment by tengqm** [comment]
@wwgfhf Please feel free to reopen this and resolve the conflicts for moving forward.

**Pull request closed without merging**